pkg/karmadactl/cmdinit/utils/examples.go

```diff
@@ -178,9 +178,9 @@ Karmada is installed successfully.
 Register Kubernetes cluster to Karmada control plane.
 
 Register cluster with 'Push' mode
-                                                                                                                                                                             
+
 Step 1: Use "%[2]s join" command to register the cluster to Karmada control plane. --cluster-kubeconfig is kubeconfig of the member cluster.
-(In karmada)~# MEMBER_CLUSTER_NAME="cat ~/.kube/config  | grep current-context | sed 's/: /\n/g'| sed '1d'"
+(In karmada)~# MEMBER_CLUSTER_NAME=%[4]scat ~/.kube/config  | grep current-context | sed 's/: /\n/g'| sed '1d'%[4]s
```

Comment:
Nice finding, what do you think of below way? It seems clearer.

```suggestion
(In karmada)~# MEMBER_CLUSTER_NAME=$(cat ~/.kube/config  | grep current-context | sed 's/: /\n/g'| sed '1d')
```

Reply:
thanks，i update it  ASAP